app/models/casa_case.rb

```diff
@@ -65,6 +69,15 @@ def update_cleaning_contact_types(args)
     end
   end
 
+  def clear_court_dates
+    
+  update(court_date: nil, 
```

Comment:
This code should be more indented, but the linter will do it the next time we run it (see the readme for lint info). :)

Reply:
It did look like something was a bit off. I had indented it earlier but at some point when making changes I may have missed that. *collapses in disgrace* Thanks for the feedback :)